### ISSUE TITLE
mailcap file is being ignored

### ISSUE THREAD

**Issue by ztx-lyghters** [paul-nameless/tg#298]
I have this set in my conf.py:
```MAILCAP_FILE = os.path.expanduser("~/.config/mailcap")```

My mailcap file has only this line:
```audio/*; vlc "%s" &```

Despite this, mp3 audio files from dialogs/groups are being opened with mpv.

**Comment by hectorBrown** [comment]
Yeah same issue here - mailcap entries seem to be irrelevant. Seems to be using [xdg-open](https://wiki.archlinux.org/title/Xdg-utils#xdg-open) though, which means you can set per-mime-type defaults with xdg-mime. Should function as a workaround for now, although you may need to make your own .desktop entries depending on the setup you want.
